demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -212,6 +218,95 @@ def is_image_path_valid(self) -> bool:
+        """
+        error_list = []
+        should_print_error = not is_pack_readme  # print error if its not pack readme
+        relative_images = re.findall(r'(\!\[.*?\])(\((?!http).*?\))', self.readme_content, re.IGNORECASE)
+        if relative_images:
+            if is_pack_readme:  # Check whether the README type is pack README which do not support relative paths.
+                for img in relative_images:
+                    try:
+                        relative_path = img[1]
+                    except IndexError:
+                        continue
+                    if relative_path:
+                        error_message, error_code = Errors.pack_readme_image_relative_path_error(img[0] + relative_path)
+                        formatted_error = self.handle_error(error_message, error_code, file_path=self.file_path,
+                                                            should_print=should_print_error)
+                        error_list.append(formatted_error)
+
+            else:  # Not pack readme relative paths are allowed but have to be valid.
+                for img in relative_images:
+                    try:
+                        relative_path = img[1]
+                    except IndexError:
+                        continue
+                    # playbook doc generator automatically adds 'Insert the link to your image here' to the
+                    # readme image path, which we want to skip.
+                    if relative_path and relative_path != '(Insert the link to your image here)':
+                        relative_path = relative_path.strip('()')
+                        if not os.path.exists(f'{str(self.pack_path)}/{relative_path}'):
+                            error_message, error_code = Errors.invalid_readme_image_relative_path_error(relative_path)
+                            formatted_error = self.handle_error(error_message, error_code, file_path=self.file_path)
+                            error_list.append(formatted_error)
+        return error_list
+
+    def check_readme_absolute_image_paths(self, is_pack_readme: bool = False) -> list:
+        """ Validate readme images absolute paths - Check if absolute paths are not broken.
+
+        Arguments:
+            is_pack_readme (bool) - True if the the README file is a pack README, default: False
+
+        Returns:
+            list: List of the errors found
+        """
+        error_list = []
+        should_print_error = not is_pack_readme  # print error if its not pack readme
+        absolute_links = re.findall(
+            r'(\!\[.*?\]|src\=)(\(|\")(https://(raw\.githubusercontent|github).com/demisto/content/.*?)(\)|\")', self.readme_content,
+            re.IGNORECASE)
+
+        if absolute_links:
+            for link in absolute_links:
+                try:
+                    img_url = link[2]
+                except IndexError:
+                    continue
+                img_url = img_url.strip('()')
```

Comment:
according to the regex, there shouldn't be anything to strip

Reply:
Done